skills/mariadb-backup-s3/scripts/validate-env.sh

```
@@ -0,0 +1,50 @@
+
+usage() {
+  echo "Usage: $0 <backup|restore|retention> [path/to/.env]"
+  echo "Validates that required env vars are present for the given command."
+  exit 1
+}
+
+cmd="${1:-}"
+env_file="${2:-.env}"
+
+[ -z "$cmd" ] && usage
+[ -f "$env_file" ] && source "$env_file"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)